test/controllers/api/auth/feeds_controller_test.rb

```diff
@@ -47,15 +47,15 @@ class << @controller; attr_accessor :prx_auth_token; end
     describe "feed tokens" do
       before do
         feed.tokens.create!(label: "something", token: "tok1")
-        feed.tokens.create!(token: "tok2")
+        feed.tokens.create!(label: "something2", token: "tok2")
```

Comment:
this and the rest of the test changes are to pass the new `:label, presence: true` validation by adding labels to all tokens where needed.

i wasn't sure if there was a need for some tokens to be unlabeled though?

Reply:
Nope - they should always have a label.  Otherwise there's no clue what apps/users might be using it.